### DISCUSSION TITLE
Remove hover underline from child element with a flex parent

### DISCUSSION THREAD

**Discussion by richstandbrook-cap** [tailwindlabs/tailwindcss#12544] [Help]
I keep coming back to this [solution for removing the underline](https://github.com/tailwindlabs/tailwindcss/discussions/3836) from a child element of a link on hover. However it doesn't work when the parent link is a flex, the display property of a child element is ignored and the hover underline is still applied. I've also tried giving the link the `group` class and the child span `group-hover:no-underline` with no luck. Anyone have a solution for this?

https://play.tailwindcss.com/sUUbzBlI04

**Comment by wongjn** [comment]
The problem comes from the fact that the underline is being applied on the `<a>` element, so even though you have `text-decoration-line: none` on the `<span>`, it has no effect. Consider applying `text-decoration-line: underline` to the other `<span>` element instead: https://play.tailwindcss.com/6wEhdDiaiX

**Reply by richstandbrook-cap** [reply]
Seems like the only option. The solution of making the child span `inline-block` works until `flex` is applied to the link. 👍
